#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-lib64.dpatch by  <root@xeno.tixteam.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Install under lib, rather than lib64, on amd64

@DPATCH@
diff -urNad xen-3.0.1+hg8743/Config.mk /tmp/dpep.2asBvw/xen-3.0.1+hg8743/Config.mk
--- xen-3.0.1+hg8743/Config.mk  2006-02-17 11:07:17.000000000 +0100
+++ /tmp/dpep.2asBvw/xen-3.0.1+hg8743/Config.mk 2006-02-18 10:42:35.000000000 +0100
@@ -27,11 +27,7 @@
 INSTALL_DATA = $(INSTALL) -m0644
 INSTALL_PROG = $(INSTALL) -m0755

-ifeq ($(XEN_TARGET_ARCH),x86_64)
-LIBDIR = lib64
-else
 LIBDIR = lib
-endif

 ifneq ($(EXTRA_PREFIX),)
 EXTRA_INCLUDES += $(EXTRA_PREFIX)/include

